lib/activity/router.js

```diff
@@ -58,7 +58,21 @@ module.exports = function route(callback) {
         // Delay GitHub API calls to avoid replication lag
         context.github.hook.before('request', avoidReplicationLag());
 
-        return callback(context, subscription, slack);
+        try {
+          await callback(context, subscription, slack);
+        } catch (err) {
+          const PERMANENT_ERRORS = [
```

Comment:
Should `is_archived` and `token_revoked` also be included in this list?
From: https://api.slack.com/methods/chat.postMessage